events/code_review_events/bugbug_utils.py

```diff
@@ -255,7 +272,11 @@ def add_new_jobs(self, revision, selected_tasks):
             return
 
         # If this diff does not belong to a revision we pushed to try, return.
-        if diff_id not in self.diff_to_push:
+        try:
+            push = self.diff_to_push.get(diff_id)
```

Comment:
NO await on get ?

Reply:
`get` only uses the memory cache for perf reasons, without accessing Redis